test/plaintime.mjs

```diff
@@ -18,255 +18,6 @@ import * as Temporal from '@js-temporal/polyfill';
 const { PlainTime, PlainDateTime } = Temporal;
 
 describe('Time', () => {
-  describe('Structure', () => {
```

Comment:
I would suggest not porting over these removals (here and in PlainDate) — these are tests that Ms2ger converted into test262-style, but I think we should keep them in the production polyfill, at least for the time being. (The idea is that the Demitasse tests will gradually disappear from the old polyfill)

Reply:
Oh, OK. I just copy/pasted the test files from the old repo because there were test failures in the new repo. I can undo the deletions and just leave the test changes. This does increase the pain of syncing any changes form old=>new, but hopefully that won't be too much going fwd?

Reply:
Yes, that sounds like the right thing to do.